File: devtools/project-core-extension-codestarts/src/main/resources/codestarts/quarkus/extension-codestarts/smallrye-graphql-codestart/java/src/main/java/org/acme/HelloGraphQLResource.java
package org.acme;

import org.eclipse.microprofile.graphql.DefaultValue;
import org.eclipse.microprofile.graphql.Description;
import org.eclipse.microprofile.graphql.GraphQLApi;
import org.eclipse.microprofile.graphql.Query;

@GraphQLApi
public class HelloGraphQLResource {

    @Query
    @Description("Say hello")
    public String sayHello(@DefaultValue("World") String name) {
        return "Hello " + name;
    }
}

File: devtools/project-core-extension-codestarts/src/main/resources/codestarts/quarkus/extension-codestarts/smallrye-graphql-codestart/kotlin/src/main/kotlin/org/acme/HelloGraphQLResource.kt
package org.acme

import org.eclipse.microprofile.graphql.DefaultValue
import org.eclipse.microprofile.graphql.Description
import org.eclipse.microprofile.graphql.GraphQLApi
import org.eclipse.microprofile.graphql.Query

@GraphQLApi
class HelloGraphQLResource {

    @Query
    @Description("Say hello")
    fun sayHello(@DefaultValue("World") name: String): String = "Hello $name"

}